modules/rabbitmq/src/main/java/org/testcontainers/containers/RabbitMQContainer.java

```diff
@@ -0,0 +1,334 @@
+        return String.format("http://" + getContainerIpAddress() + ":" + getMappedPort(DEFAULT_HTTP_PORT));
+    }
+
+    /**
+     * @return URL of the HTTPS management endpoint.
+     */
+    public String getHttpsUrl() {
+        return String.format("https://" + getContainerIpAddress() + ":" + getMappedPort(DEFAULT_HTTPS_PORT));
+    }
+
+    /**
+     * Sets the password for the admin (default is <pre>guest</pre>)
+     *
+     * @param adminPassword The admin password.
+     * @return This container.
+     */
+    public RabbitMQContainer withAdminPassword(final String adminPassword) {
+        this.adminPassword = adminPassword;
+        return this;
+    }
+
+    public enum SslVerification {
+        VERIFY_NONE("verify_none"), VERIFY_PEER("");
+
+        SslVerification(String value) {
+            this.value = value;
+        }
+
+        private final String value;
+    }
+
+    public RabbitMQContainer withSSL(
+            final MountableFile keyFile,
+            final MountableFile certFile,
+            final MountableFile caFile,
+            SslVerification verify,
+            boolean failIfNoCert,
+            int verificationDepth) {
+
+        return withSSL(keyFile, certFile, caFile, verify, failIfNoCert)
+                .withEnv("RABBITMQ_SSL_DEPTH", String.valueOf(verificationDepth));
+    }
+
+    public RabbitMQContainer withSSL(
+            final MountableFile keyFile,
+            final MountableFile certFile,
+            final MountableFile caFile,
+            SslVerification verify,
+            boolean failIfNoCert) {
+
+        return withSSL(keyFile, certFile, caFile, verify)
+                .withEnv("RABBITMQ_SSL_FAIL_IF_NO_PEER_CERT", String.valueOf(failIfNoCert));
+    }
+
+    public RabbitMQContainer withSSL(
+            final MountableFile keyFile,
+            final MountableFile certFile,
+            final MountableFile caFile,
+            SslVerification verify) {
+
+        return withEnv("RABBITMQ_SSL_CACERTFILE", "/etc/rabbitmq/ca_cert.pem")
+                .withEnv("RABBITMQ_SSL_CERTFILE", "/etc/rabbitmq/rabbitmq_cert.pem")
+                .withEnv("RABBITMQ_SSL_KEYFILE", "/etc/rabbitmq/rabbitmq_key.pem")
+                .withEnv("RABBITMQ_SSL_VERIFY", verify.value)
+                .withCopyFileToContainer(certFile, "/etc/rabbitmq/rabbitmq_cert.pem")
+                .withCopyFileToContainer(caFile, "/etc/rabbitmq/ca_cert.pem")
+                .withCopyFileToContainer(keyFile, "/etc/rabbitmq/rabbitmq_key.pem");
+    }
+
+    public RabbitMQContainer withPluginsEnabled(String... pluginNames) {
+        List<String> command = new ArrayList<>(asList("rabbitmq-plugins", "enable"));
+        command.addAll(asList(pluginNames));
+        values.add(command);
+        return self();
+    }
+
+    public RabbitMQContainer withBinding(String source, String destination) {
+        values.add(asList("rabbitmqadmin", "declare", "binding", "source=" + source, "destination=" + destination));
```

Comment:
Sorry for the super-picky comment, but please could you adjust the formatting of these methods a bit? It feels like it might be easier to read if each of the key-value pairs was on its own line
e.g. 
```suggestion
        values.add(asList("rabbitmqadmin", "declare", "binding", 
        	"source=" + source, 
        	"destination=" + destination
        ));
```

And similarly for the other methods.

Reply:
Fully agree, that's a good suggestion! 👍

Reply:
Done